### PR TITLE
Removed </a> tag that we don't need

### PR DESCRIPTION
**Pull request by naslundx**

### CODE REVIEW

**Comment by naslundx** [comment]
Great job!

**Comment by Intergalaxen** [comment]
good job!

File: views/pages/index.ejs [hunk 1]
```diff
@@ -16,7 +16,7 @@
     <h3>Meddelanden:</h3>
     <ul>
         <% messages.forEach(function(msg) { %>
-        <li><%= msg %></a></li>
```
Comment (naslundx) [inline views/pages/index.ejs:19]: Oops! Good catch

**Comment by naslundx** [comment]
Could you please also add something to the top of the page?

**Pull request merged**